stubs/enterReplies.js

```diff
@@ -1 +1,27 @@
-module.exports = ['Hi', 'Target Acquired', 'Firing', 'Hello friend.', 'Gotcha', 'I see you']
\ No newline at end of file
+introduction = 'Please join hackbot in welcoming @user to the Rogue Hack Lab!!! :tada:';
+
+icebreakers = [
+  'What\'s your favorite 1990\'s cartoon?',
+  'What did you have for breakfast?',
+  'What\'s one skill you would like to develop?',
+  'What\'s your favorite letter of the alphabet and why?',
+  'How old were you when you first used a computer?',
+  'What\'s your favorite spot for a beer?',
+  'How did you hear about the hack lab?',
+  'When was the last time you changed your hairstyle?',
+  'Have you committed today?',
+  'You\'re trapped on a deserted island, what TV show are you bringing to keep you company?',
+  'What\'s your favorite flavor of sparkling water?',
+  'Tell us your favorite \'nerdy\' joke',
+  'How old were you when you stopped believing in Santa Claus?',
+  'Share your favorite color (in hexcode!)',
+  'What website do you suddenly find yourself on when you\'re \"just checking your email\"?',
+  'If you could be any other species, which would you choose?',
+  'What superpower would you want?',
+  'What\'s your patronus?'
+];
+
+// Combine introduction with an icebreaker question
+responses = icebreakers.map((item) => introduction + ' ' + item + 'Start a thread with your response to get to know the hack lab better');
```

Comment:
This approach is nice and simple however these are used for enter/exit from all rooms hackbot is in. If you change that to filter to only general I love it. Also we could make hackbot do the introduction then start a thread on the message it just posted with the comment. Would likely need to @ the user.

Reply:
Thinking about this you might need to get running with a test bot to get the right way to tell the channel. Let me know if you want to pair on that.

Reply:
Thanks Pat! I'd love to get a test bot up and running and be able to do some more thorough testing. I'll read up on how to narrow down the messages. My initial feeling is to just filter the messages in this function, and use some of the older style short messages for other room arrivals. I don't think I've seen many room exits, so I'll need to dig into that more.